main.go

```diff
@@ -72,6 +73,19 @@ func main() {
 	mongoDBAuth := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_AUTH", true, true)
 	mongoDBName := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_DATABASE", true, false)
 	mongoTimeout := envLoader.GetAndLogEnvVar("ROKWIRE_CORE_MONGO_TIMEOUT", false, false)
+	// webhook configs
+	githubWebhookRequestToken := envLoader.GetAndLogEnvVar("GITHUB_WEBHOOK_REQUEST_TOKEN", false, false)
+	githubToken := envLoader.GetAndLogEnvVar("GITHUB_TOKEN", false, false)
+	githubOrgnizationName := envLoader.GetAndLogEnvVar("GITHUB_ORG_NAME", false, false)
+	githubRepoName := envLoader.GetAndLogEnvVar("GITHUB_REPO_NAME", false, false)
+	githubWebhookConfigPath := envLoader.GetAndLogEnvVar("GITHUB_WEBHOOK_CONFIG_PATH", false, false)
```

Comment:
We should add another env var for `GITHUB_APP_CONFIG_BRANCH` and filter out any commits that are not made to the specified branch. This is because we may want to use different branches for different environments, or we may want to have staging branches that do not actually trigger an update.